src/commands/community/prune/whitelist.ts

```diff
@@ -40,20 +40,28 @@ export async function getExcludedRoles(guild: Guild): Promise<Role[]> {
 }
 
 export async function whitelistRolesEmbed(roles: Role[]) {
+  if (roles.length == 0) {
```

Comment:
```suggestion
  if (roles.length === 0) {
```

Best practice to always use `===`